book-example/src/format/config.md

```diff
@@ -59,10 +62,8 @@ This controls the build process of your book.
   will be created when the book is built (i.e. `create-missing = true`). If this
   is `false` then the build process will instead exit with an error if any files
   do not exist.
-- **preprocess:** Specify which preprocessors to be applied. Default is
```

Comment:
We'll need to mention the `use-default-preprocessors` flag here. It's used to enable/disable including the default preprocessors and defaults to `true`.

I've also added a couple tests in 18a36ec to make sure the `build.use-default-preprocessors` flag works as intended.